### DISCUSSION TITLE
Wazuh Still Listening on Port 514 After Removing <remote> Configurations

### DISCUSSION THREAD

**Discussion by karimbeltesing** [wazuh/wazuh#26981] [Q&A] [answered]
Hello everyone,

I recently stopped receiving logs from FortiClient on my Wazuh server. Initially, it was working fine, but now no logs are showing when I check using: `tail -f /var/ossec/logs/archives/archives.log`

Here’s my Wazuh server configuration for receiving syslogs:
```
<remote>
  <connection>syslog</connection>
  <port>514</port>
  <protocol>udp</protocol>
  <allowed-ips>my endpoint address</allowed-ips>
  <local_ip>Wazuh server</local_ip>
</remote>
```
Troubleshooting Steps:
1. Wireshark on my endpoint confirms that logs are leaving.
2. TCPDump on the Wazuh server shows the logs arriving on UDP port 514.
3. Despite this, no logs appear in the archives.log file.

**Additional Findings:**
Using the following command: `cat /var/ossec/logs/ossec.log | grep 514`
I found this error: `CRITICAL: (1206): Unable to Bind port '514' due to [(98)-(Address already in use)]`
I’m unsure how to resolve this error as I only have one address configured in /var/ossec/etc/ossec.conf to send logs to port 514 via syslog.

Then I removed all the <remote> tags from /var/ossec/etc/ossec.conf and restarted the wazuh-manager service. As expected, the error no longer appears.
However, when I run netstat -tulnp, I see that Wazuh is still listening on port 514, with the server's IP address:
`udp   200192      0 X.X.X.X:514        0.0.0.0:*                           224304/wazuh-remote`

**Question:
Why is Wazuh still listening on port 514 even though all <remote> configurations have been removed?**

Thank you for your insights and assistance!

**Comment by SirStephanikus** [comment, marked as answer]
Hello Karim

It sounds more like a Linux specific issue, and not Wazuh.

However:
Do this:

Reboot fully ➜ then switch as root: 
Instead of using `netstat`, use `ss -tulpen | grep -i 514`, then comeback with the results.

Additionally, I suggest switching to syslog TCP, and not UDP. TCP can usually receive bigger syslog messages.

**Reply by karimbeltesing** [reply]
you're right it closed after I restarted the server.
Thank you for your response.

**Comment by karimbeltesing** [comment]
Problem solved, Closing discussion. Thank you @SirStephanikus for the help
